src/Components/Search/NewSearch/SuggestionItem/NewSuggestionItem.tsx

```diff
@@ -2,14 +2,14 @@ import * as React from "react"
 import { FC } from "react"
 import { SuggestionItemLink } from "./SuggestionItemLink"
 import { DefaultSuggestion } from "./DefaultSuggestion"
-import { QuickNavigation } from "./QuickNavigation"
+import { Box, Pill } from "@artsy/palette"
+import { QuickNavigationItem } from "Components/Search/NewSearch/SuggestionItem/QuickNavigationItem"
 
 export interface SuggestionItemOptionProps {
   text: string
   value: string
   subtitle: string
   imageUrl: string
-  showArtworksButton: boolean
```

Comment:
i dont fully follow the removal of the prop, but I believe it

Reply:
So since we restructured the artist tabs; the artworks tab is now the 'first' so this button was redundant.

Reply:
Selecting the normal result will always go to the artworks